### DISCUSSION TITLE
Future plans for Vue

### DISCUSSION THREAD

**Discussion by z0091** [tailwindlabs/headlessui#3541] [Help]
Hello Headless UI team,

I was wondering if there are any plans to actively maintain and develop the Vue version of the Headless UI package. It seems that the React version has progressed significantly, with v2 already featuring multiple enhancements. However, I don't see similar changes or improvements being made for the Vue version.

Could you clarify if the Vue version will receive the same attention and updates? Are there any roadblocks or different priorities affecting the release of v2 for Vue? Knowing your roadmap for Vue support would help us make informed decisions on whether to continue using Headless UI in our Vue projects.

Thank you for your hard work and dedication! Looking forward to your response.

**Comment by dennis-gonzales** [comment]
+1

**Comment by weirdlooop** [comment]
+1 we use headless UI for vue and are awaiting an update regarding v2... 

Thanks for the hardwork

**Comment by Baroshem** [comment]
Would like to know as well! :)

**Comment by dikaso** [comment]
See https://github.com/tailwindlabs/headlessui/discussions/3183#discussioncomment-9941207. No plans for  2024.

**Reply by hemju** [reply]
Since it is now 2025, maybe there are updated plans for Vue support and Headless V2.

> Knowing your roadmap for Vue support would help us make informed decisions on whether to continue using Headless UI in our Vue projects.

Exactly, if it is simply maintenance, then we will keep it in existing projects. If it is abandoned, we have to think about alternatives or the best case; it will receive support in the future and keep it and use it in our projects.

Can we have an update on the roadmap? @RobinMalfait @adamwathan 

Reference:
https://github.com/tailwindlabs/headlessui/discussions/3183
